sources/packages/backend/libs/services/src/report/report.service.ts

```diff
@@ -4,6 +4,8 @@ import { RecordDataModelService, ReportConfig } from "@sims/sims-db";
 import { CustomNamedError, StringBuilder } from "@sims/utilities";
```

Comment:
Please remove the unused import. 

![image](https://github.com/bcgov/SIMS/assets/54600590/de7ed745-bb31-4b28-a608-17cb68e2f6cb)

Reply:
Removed.